lib/curation_concerns/engine.rb

```diff
@@ -18,6 +18,7 @@ class Engine < ::Rails::Engine
     require 'breadcrumbs_on_rails'
 
     config.autoload_paths += %W(
+      #{config.root}/app/actors/concerns
```

Comment:
This matches my understanding of autoload, which is that Rails will handle `app/*/*.rb` and will handle the `concerns` subdirectory in directories it knows about (models, controllers -- not actors), but that it won't recursively load everything in `app/`.